module/VuFind/tests/unit-tests/src/VuFindTest/Validator/LaminasTokenCountingCsrfTest.php

```diff
@@ -39,7 +39,7 @@
  * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
  * @link     https://vufind.org/wiki/development:testing:unit_tests Wiki
  */
-class CsrfTest extends \PHPUnit\Framework\TestCase
+class LaminasTokenCountingCsrfTest extends \PHPUnit\Framework\TestCase
```

Comment:
This test class should be renamed to match the renaming of the class under test.